### DISCUSSION TITLE
Template creation

### DISCUSSION THREAD

**Discussion by CptHowdy2012** [docmost/docmost#221] [Ideas]
Add ability to turn a page into a template and a menu item when you click "..." to create page from template. Seems to be a simple improvement that would help save time when entering many pages that are structured similarly.

**Comment by lephtHanded** [comment]
or just the ability to copy a page and paste it elsewhere.  (just  have one folder for templates and copy/paste them where needed)
![image](https://github.com/user-attachments/assets/ba3338be-4e1d-45e1-bf70-be9db6325eb1)

**Comment by WavedirectTel** [comment]
I think that is doable and I think that is a good idea.  Would save a little bit of work because I'm not sure you could copy and paste a page with embeds and such.

**Comment by lephtHanded** [comment]
What I ended up doing was exporting my template page as HTML and just re-importing it and changing it's name to what I needed it to be.
